npm/pkg/dataplane/policies/chain-management_linux.go

```diff
@@ -419,6 +441,45 @@ func (pMgr *PolicyManager) positionAzureChainJumpRule() error {
 	return nil
 }
 
+// add the jump from OUTPUT chain to AZURE-NPM chain if it doesn't exist
+func (pMgr *PolicyManager) appendAzureChainJumpRuleFromOutputChain() error {
```

Comment:
could you generalize `positionAzureChainJumpRule()` to take in arguments for the chain name and placeFirst boolean?

Reply:
updated